vhs/vhs-test.js

```diff
@@ -0,0 +1,79 @@
+const { describe } = require("mocha");
+var {createTape, reset, createCollection, previewCollection} =  require("./vhs")
+var assert = require('chai').assert;
+
+describe("Tape", function() {
+
+  it.skip("should create tape", function() {
+    var genericTape = createTape("",false)
+   
+    assert.deepEqual(genericTape, {title:"", readyToPlay: false})
+  })
+
+  it.skip("should have a title", function(){
+    var littleMermaid = createTape("The Little Mermaid", false)
+    var landBeforeTime = createTape("The Land Before Time", false)
+
+    assert.equal(littleMermaid.title,"The Little Mermaid")
+    assert.equal(landBeforeTime.title,"The Land Before Time")
+  })
+
+  it.skip("should have a readyToPlay status by default", function() {
+    var littleMermaid = createTape("The Little Mermaid", false);
```

Comment:
To pass this test, nothing has to be changed from previous tests. I wonder if the second argument was meant to be left out when calling createTape for this one so that students are forced to create a default value?